### PR TITLE
Fix Intellisense in VS

### PR DESCRIPTION
**Pull request by karajas**
/cc @weshaggard 

Resolves https://github.com/dotnet/corefx/issues/15253

### CODE REVIEW

File: targetingpacks.props [hunk 1]
```diff
@@ -23,6 +23,10 @@
       </Reference>
       <ReferencePath Include="@(ReferenceFromRuntime->'$(RuntimePath)%(Identity).dll')" />
     </ItemGroup>
+    <!-- Include references for VS -->
+    <ItemGroup Condition="'$(DesignTimeBuild)'=='true'">
```
Comment (weshaggard) [inline targetingpacks.props]: How does this help anything? You are setting the item group to the same value.

Comment (karajas) [inline targetingpacks.props]: Pushed the wrong branch, will correct shortly.

**Comment by karelz** [comment]
@karajas what's the next step for this PR?

**Comment by danmosemsft** [comment]
@karajas ping?

**Comment by karelz** [comment]
@karajas ping?

**Comment by karajas** [comment]
I'm not sure these changes are relevant anymore. I will close them out for now.

**Comment by weshaggard** [comment]
@karajas please make sure we at least test to see if it is still broken or not.

**Comment by karajas** [comment]
Building on latest master to verify. From testing out a few Sln files, it looks like the issue no longer persists. I've opened it up in both 2015 and 2017 RC.
I'm not sure exactly how the issue resolved itself, will investigate later today.

**Comment by karajas** [comment]
This refactoring of the references seems to have resolved the issue https://github.com/dotnet/corefx/commit/6cb8c638f4d49b8f723c8012707c1dca57dadd54

**Pull request closed without merging**